### DISCUSSION TITLE
[Feature] does skywalking support hot update of MAL or lal now in2024?

### DISCUSSION THREAD

**Discussion by taoyang987** [apache/skywalking#11864] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

https://github.com/apache/skywalking/issues/6897
https://github.com/apache/skywalking/pull/7039
in 2021,@ @dmsolr   @wu-sheng  said sw did't support.
now in2024,does skywalking support hot update of MAL or lal?

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
AFAIK, we don't do, because OAP running as a cluster in product env, update codes in a distributed system is not good.
